SwanContents/setup.py

```diff
@@ -36,15 +34,13 @@
     url="https://github.com/swan-cern/jupyter-extensions",
     author="SWAN Admins",
     description="SWAN Contents Manager for Jupyter",
-    long_description= long_description,
+    long_description=long_description,
```

Comment:
These are all format changes, but ok... let's just get done with it.

Reply:
Some are, others aren't. I applied the formatter to files that were being changed.